pkg/controller/operators/olm/operator_test.go

```diff
@@ -3353,7 +3347,7 @@ func TestSyncOperatorGroups(t *testing.T) {
 		},
 	}
 	deploymentName := "csv1-dep1"
-	crd := crd("c1", "v1", "fake.api.group")
+	crd := crd("CustomResourceDefinition", "v1beta1", "apiextensions.k8s.io")
```

Comment:
this is confusing - I assume this change is to make the test cases that check for CRDs match, but what this is actually doing is defining a CSV that owns a CRD that defines an API called CRD. Can you keep these values the same, and update all of the expected statuses to the right values instead?